projects/kit/components/input-phone-international/input-phone-international.style.less

```diff
@@ -0,0 +1,50 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+:host {
+    display: block;
+
+    &._disabled {
+        pointer-events: none;
+    }
+}
+
+.t-country-select {
+    width: 3.625rem;
+
+    &[data-size='m'] {
+        width: 3.5rem;
+    }
+
+    &[data-size='s'] {
+        width: 2.75rem;
+    }
+}
+
+.t-input-phone {
+    flex: 1;
+}
+
+.t-flag {
+    width: 1.75rem;
+    height: 1.25rem;
+
+    &_select {
+        .fullsize(absolute, inset);
+
+        left: 0.5rem;
+        margin: auto;
+    }
+
+    :host(._disabled) & {
+        opacity: var(--tui-disabled-opacity);
+    }
+}
+
+.t-country-item-name {
+    margin-inline: 0.75rem auto;
```

Comment:
Not supported by Safari in a shorthand form:
https://caniuse.com/?search=margin-inline

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/7580/commits/a6e889c5c2109ef32eb045d9f560acacc8c584aa